gensim/models/base_any2vec.py

```diff
@@ -130,7 +130,27 @@ def _do_train_epoch(self, corpus_file, thread_id, offset, cython_vocab, thread_p
         raise NotImplementedError()
 
     def _do_train_job(self, data_iterable, job_parameters, thread_private_mem):
-        """Train a single batch. Return 2-tuple `(effective word count, total word count)`."""
+        """Train a single batch.
+
+        Parameters
+        ----------
+        data_iterable : an iterable chunk of data representing a batch
```

Comment:
Please replace "object" by the appropriate type.

```suggestion
        data_iterable: object
          an iterable chunk of data representing a batch
```

and do the same for the lines below.